src/ray/util/common_runtime/test/CMakeLists.txt

```diff
@@ -1,13 +1,21 @@
 SET(CMAKE_CXX_STANDARD 11)
 
+include(../../../../../thirdparty/scripts/thirdparty.cmake)
 
-include(../CMakeLists.txt)
+find_package(GTest)
+include_directories(${GTEST_INCLUDE_DIR})
+link_directories(${GTEST_LIBS})
 
 include_directories(../)
+link_directories(../)
```

Comment:
We can omit this as inside cmakefile projects, the link path is automatically resolved